core/processor.go

```diff
@@ -86,14 +88,20 @@ func (p *Processor) Start() {
 			// Solana
 			watcher = solana.NewWatcher(cfg, p.db, p.txsCh, p.txTrackCh)
 			dispatcher = solana.NewDispatcher(cfg.Rpcs, cfg.Wss)
+		} else if chain == "lisk-testnet" {
+			client := chainlisk.NewLiskClient(cfg)
+			watcher = chainlisk.NewWatcher(p.db, cfg, client)
+
 		} else {
 			panic(fmt.Errorf("Unknown chain %s", chain))
 		}
 
 		p.watchers[chain] = watcher
 		go watcher.Start()
-		p.dispatchers[chain] = dispatcher
-		dispatcher.Start()
+		if dispatcher != nil {
```

Comment:
There is no nil checking. Create an empty dispatcher in Lisk and remove this nil checking. The app is supposed to crash if there is no dispatcher